src/config/feature-manager/types.ts

```diff
@@ -80,6 +80,11 @@ export interface IFeatureManager {
   [FeatureNames.WebhookVerifiedContent]: IWebhookVerifiedContent
 }
 
+export interface RegisteredFeature<T extends FeatureNames> {
+  isEnabled: boolean
+  props?: IFeatureManager[T]
```

Comment:
why is props optional? Isn't it always defined for all features?

Reply:
It's only defined if the feature is enabled, so there is still a chance it is undefined

Reply:
you are right.